ui/src/saros/stf/server/rmi/htmlbot/impl/HTMLBotImpl.java

```diff
@@ -36,13 +36,13 @@ public IRemoteHTMLView view(View view) {
 
   @Override
   public List<String> getAccountList() throws RemoteException {
-    return BotUtils.getListItemsText(getBrowser(MainPage.class), SELECTOR_ACCOUNT_ENTRY);
+    return new JQueryHelper(getBrowser(MainPage.class)).getListItemsText(SELECTOR_ACCOUNT_ENTRY);
   }
 
   @Override
   public List<String> getContactList(View view) throws RemoteException {
-    return BotUtils.getListItemsText(
-        getBrowser(view.getPageClass()), SELECTOR_CONTACT_ITEM_DISPLAY_NAME);
+    return new JQueryHelper(getBrowser(view.getPageClass()))
```

Comment:
Same as in `getAccountList()`. Move instantiation to CTOR and hold as instance variable?